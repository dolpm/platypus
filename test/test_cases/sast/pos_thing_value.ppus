thing MyThing <| {
  im_an_int: int
}

pipe main [] |> unit {
  MyThing t = MyThing {
    im_an_int: 5
  };

  |> ();
}