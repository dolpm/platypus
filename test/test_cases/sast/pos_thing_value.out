thing MyThing = {
im_an_int: int
}pipe main [] |> unit {
{(id: main_with_args)
{(id: main_wrapper)
  MyThing t = (MyThing : MyThing {
    im_an_int: (int : 5)
  });
  |> (unit : ());
}
}
}
